vmdb/app/assets/javascripts/cfme_application.js

```diff
@@ -1237,3 +1238,8 @@ function miqJqueryRequest(url, options) {
 
   new $j.ajax(options['no_encoding'] ? url : encodeURI(url), ajax_options);
 }
+
+function miqDomElementExists(element){
+  return $j('#' + element).length > 0 ? true : false
```

Comment:
Do you need the `? true : false`.  I would expect `.length > 0` to return a boolean.